packages/core/__tests__/utils/__integration__/__scripts__/exitOnFailure.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# function to exit if we encounter a bad return code
+exitOnFailure () {
```

Comment:
yeah, this could definitely be reused if exposed at a higher level than just `cli/auth/system` or `core/utils/integration` 😋